test/load.test.js

```diff
@@ -67,3 +67,15 @@ describe('crap load', function () {
   });
 });
 
+it('should load an entire chain of CRaP without config', function (done) {
```

Comment:
Can you change the description of this test because it isn't without config, The configs are just local to the components. Something like "should load an entire chain of CRaP using local configs" would work.